consensus/common_test.go

```diff
@@ -317,67 +317,138 @@ func ensureNoNewEvent(ch <-chan interface{}, timeout time.Duration,
 	}
 }
 
-func ensureNoNewStep(stepCh <-chan interface{}) {
+func ensureNoNewEventOnChannel(ch <-chan interface{}) {
+	ensureNoNewEvent(ch, ensureTimeout, "We should be stuck waiting, "+
+		"not receiving new event on the channel")
+}
+
+func ensureNoNewRoundStep(stepCh <-chan interface{}) {
 	ensureNoNewEvent(stepCh, ensureTimeout, "We should be stuck waiting, "+
-		"not moving to the next step")
+		"not receiving NewRoundStep event")
+}
+
+func ensureNoNewUnlock(unlockCh <-chan interface{}) {
+	ensureNoNewEvent(unlockCh, ensureTimeout, "We should be stuck waiting, "+
+		"not receiving Unlock event")
 }
 
 func ensureNoNewTimeout(stepCh <-chan interface{}, timeout int64) {
 	timeoutDuration := time.Duration(timeout*5) * time.Nanosecond
 	ensureNoNewEvent(stepCh, timeoutDuration, "We should be stuck waiting, "+
-		"not moving to the next step")
+		"not receiving NewTimeout event")
 }
 
-func ensureNewEvent(ch <-chan interface{}, timeout time.Duration, errorMessage string) {
+func ensureNewEvent(ch <-chan interface{}, height int64, round int,
+	timeout time.Duration, errorMessage string) {
```

Comment:
Should be broken up by param per line with trailing comma and newline behind the last one.